scipy/stats/tests/test_distributions.py

```diff
@@ -2712,6 +2712,49 @@ def test_skewcauchy_R(self):
         assert_allclose(stats.skewcauchy.ppf(cdf, a), x)
 
 
+# Test data for TestSkewNorm.test_noncentral_moments()
+# The expected noncentral moments were computed by Wolfram Alpha.
+# In Wolfram Alpha, enter
+#    SkewNormalDistribution[0, 1, a] moment
+# with `a` replaced by the desired shape parameter.  In the results, there
+# should be a table of the first four moments. Click on "More" to get more
+# moments.  The expected moments start with the first moment (order = 1).
+_skewnorm_noncentral_moments = [
```

Comment:
Same here, this will incur a cost at the test collection phase, even if you are doing `pytest scipy -k test_firwin` or so. You could instead make this a `pytest.fixture(scope='module')` that returns these values, then use it inside the function. Then the cost is only incurred during setup of the specific test(s) that need these values.

Reply:
Can a fixture be used as an argument to `pytest.mark.parametrize()`?

Reply:
Apparently that's an old FAQ: https://github.com/pytest-dev/pytest/issues/349

Reply:
@larsoner, I'm punting on the fixture stuff.  If you know exactly what you want to do, perhaps you could do the optimization in a follow-up PR, or just push your change to this PR?